flow/benchmarks/run_all_benchmarks.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
nice! can we document this somewhere though? (either a the start of this script or in the benchmarks' `README.md`)

Reply:
peep the README.md